### ISSUE TITLE
Add create/browse filter FieldOccurences to TW

### ISSUE THREAD

**Issue by tmcelrath** [SpeciesFileGroup/taxonworks#1944]
As a user, I want to store data on Field Occurrences or "Living Specimens" e.g. Term Name dwc:basisOfRecord - basically, things that are not Collection Objects, such as occurrences that are not taken as a PhysicalSpecimen. 

These would have most of the same facets as a Collection Object, except without associated fields specific to COs, such as preparation, type status (although this is arguable viz. species described from photos), catalog number, etc ... but would include information about a collecting event, determinations, etc ...

**Comment by proceps** [comment]
same as #1643?

**Comment by mjy** [comment]
Yes. https://github.com/SpeciesFileGroup/taxonworks/issues/1643.  We could alter this to be the corresonding task.
